lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -46,7 +46,17 @@ class GoogleCloudOutput < BufferedOutput
     CLOUDFUNCTIONS_SERVICE = 'cloudfunctions.googleapis.com'
```

Comment:
Have you built a gem with this version already? Do we need to bump the version again?